src/api/verify.ts

```diff
@@ -103,7 +116,9 @@ class VerifyAPI {
 
   static async startTrackingVerified(params: RadarStartTrackingVerifiedParams) {
     const doTrackVerified = async () => {
-      const trackRes = await this.trackVerified({});
+      const trackRes = await this.trackVerified({
+        ...params,
```

Comment:
nit: can probably just pass `params` instead of destructuring

Reply:
Good call, updated